### PR TITLE
Rename "shift*" to "reshuffle"

### PR DESCRIPTION
**Pull request by arielshaqed**
Most changes are mechanical, unfortunately there are many of them.

Tested by "git grep" in many forms (not saying which, so brave reviewers can suggest something I may have missed).

Unsure how to update .circleci (where are the creds stored?), so not doing anything there yet.

### CODE REVIEW

File: app-testsuite/verdaccio.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ uplinks:
     url: https://registry.npmjs.org/
 
 packages:
-  'shift-cli':
+  '@reshuffle/*':
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply app-testsuite/verdaccio.yaml]: Intentional product decision.  The CLI is `reshuffle`, the packages are under `@reshuffle`.

File: cli/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Change Log - shift-cli
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply cli/CHANGELOG.md]: Good idea!  So I tried it, but `rush change` _doesn't_ touch this title.  This sort-of makes sense for auto-generated changelogs.

File: cli/package.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "shift-cli",
-  "description": "ShiftJS CLI Tool",
+  "name": "reshuffle",
+  "description": "Reshuffle CLI Tool",
   "version": "0.2.3",
   "author": "arikmaor @arikmaor",
   "bin": {
-    "shift-cli": "./bin/run"
+    "": "./bin/run"
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply cli/package.json]: Good catch!  Looks like a bug.  Fixed.

File: cli/package.json [hunk 4]
```diff
@@ -16,22 +16,22 @@
     "/npm-shrinkwrap.json",
     "/oclif.manifest.json"
   ],
-  "homepage": "https://github.com/binaris/shiftjs",
+  "homepage": "https://www.reshuffle.com",
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply cli/package.json]: Changed to (new) GitHub.

File: install_script.sh [hunk 5]
```diff
@@ -7,25 +7,25 @@ if ! grep -q 'react-scripts eject' package.json; then
 fi
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply install_script.sh]: Not done, that can go in a separate PR.

**Comment by bergundy** [comment]
@arielshaqed I merged #221, I can help you with the rebase and renaming.

**Comment by arielshaqed** [comment]
Branch passing: https://ci.binaris.io/blue/organizations/jenkins/gobinaris%2Fprecious/detail/master/8720/pipeline/369/

**Comment by arielshaqed** [comment]
> Thanks!
> 
> Went over all the changes, can be considered approved when integration passes.

@vogre could you please (approve)?

File: code-transform/src/index.ts [hunk 6]
```diff
@@ -11,7 +11,7 @@ function insertExpose(path: NodePath<BabelTypes.Node>, t: typeof BabelTypes, fun
   path.insertAfter(
     t.expressionStatement(
       t.assignmentExpression('=',
-        t.memberExpression(t.identifier(funcName), t.identifier('__shiftjs__')),
+        t.memberExpression(t.identifier(funcName), t.identifier('__visibility__')),
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply code-transform/src/index.ts]: Done!

**Comment by arielshaqed** [comment]
Pull with binaris/valet#411 and binaris/spice#666.  Yes, both are typological numbers.

**Comment by arielshaqed** [comment]
Branch passed integration on https://ci.binaris.io/blue/organizations/jenkins/gobinaris%2Fprecious/detail/master/8726/pipeline/373/

**Pull request merged**